### ISSUE TITLE
Make a Call between 2 Android Clients

### ISSUE THREAD

**Issue by mzoloski** [pchab/AndroidRTC#104]
Hello guys,

I'm trying to make a call between 2 android clients... and don't have any result.
I succeed in sending datas betweens these clients. I just emit from AndroidRTC to ProjectRTC, and when i received from ProjectRTC, i emit again to the second AndroitRTC.
Should i do the same way ? I didnt succeed like this...
Anyone succeed for doing this there ? Any help will be deeply appreciated.

Marc

**Comment by BackPackerDz** [comment]
Hi ! did you add the ProjectRTC to your server ?

**Comment by mzoloski** [comment]
Yes. You already succeed to send stream from one Android device to another ?

**Comment by BackPackerDz** [comment]
Yes !

**Comment by mzoloski** [comment]
Can you send me your code or help me ? What did you change in your clients ?
